examples/vision/shiftvit.py

```diff
@@ -784,11 +839,25 @@ def __call__(self, step):
             step > self.total_steps, 0.0, learning_rate, name="learning_rate"
         )
 
+    def get_config(self):
```

Comment:
Added this to avoid following error while saving model:
_"NotImplementedError: Learning rate schedule must override get_config"_